app/src/main/res/layout/weight_record_fragment.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
+    <data>
+
+        <variable
+            name="weightRecordViewModel"
+            type="com.example.weightcontrolapp.ui.weightrecord.WeightRecordViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
```

Comment:
Make the text font bold, text colour black.

Reply:
[ee17e5c](https://github.com/kenta7777/WeightControlApp/pull/1/commits/ee17e5cfdc52e44e453fbb26e6cc82fd2a3552db)